WalletWasabi.Fluent/ViewModels/Wallets/Home/History/Details/TransactionDetailsViewModel.cs

```diff
@@ -17,27 +17,27 @@ namespace WalletWasabi.Fluent.ViewModels.Wallets.Home.History.Details;
 public partial class TransactionDetailsViewModel : RoutableViewModel
 {
 	private readonly WalletViewModel _walletVm;
-
-	[AutoNotify] private bool _isConfirmed;
-	[AutoNotify] private int _confirmations;
-	[AutoNotify] private int _blockHeight;
-	[AutoNotify] private string? _dateString;
-	[AutoNotify] private Money? _amount;
-	[AutoNotify] private LabelsArray? _labels;
-	[AutoNotify] private string? _transactionId;
-	[AutoNotify] private string? _blockHash;
+	[AutoNotify] private BtcAmount? _amount;
 	[AutoNotify] private string? _amountText = "";
+	[AutoNotify] private string? _blockHash;
+	[AutoNotify] private int _blockHeight;
+	[AutoNotify] private int _confirmations;
 	[AutoNotify] private TimeSpan? _confirmationTime;
+	[AutoNotify] private string? _dateString;
 	[AutoNotify] private bool _isConfirmationTimeVisible;
+
+	[AutoNotify] private bool _isConfirmed;
 	[AutoNotify] private bool _isLabelsVisible;
+	[AutoNotify] private LabelsArray? _labels;
+	[AutoNotify] private string? _transactionId;
 
 	private TransactionDetailsViewModel(TransactionSummary transactionSummary, WalletViewModel walletVm)
 	{
 		_walletVm = walletVm;
 
 		NextCommand = ReactiveCommand.Create(OnNext);
 
-		Fee = transactionSummary.Fee;
+		Fee = BtcAmount.Create(transactionSummary.Fee ?? Money.Zero);
```

Comment:
It might be nicer to handle the null value inside of the `BtcAmount` class. The same way as you did here.
(Instead of doing ?? Money.Zero everywhere)